packages/rx_bloc_cli/CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 ## [4.0.0]
```

Comment:
Let's release the MFA and eco system breaking changes as a separate releases. With that being said feel free to use the version 5.0.0 for the breaking changes related issues